index.js

```diff
@@ -46,6 +46,10 @@ var PhantomJSBrowser = function(baseBrowserDecorator, config, args, logger) {
       }));
     }
 
+    if (config && config.exitOnResourceError) {
+      optionsCode.unshift('page.onResourceError = function() { phantom.exit(1); };');
```

Comment:
Could you move this into a template file, like we do with the debug code? I really don't like having code in strings